src/components/TextInput/BaseTextInput.js

```diff
@@ -371,6 +371,7 @@ function BaseTextInput(props) {
                 </PressableWithoutFeedback>
                 {!_.isEmpty(inputHelpText) && (
                     <FormHelpMessage
+                        style={props.hintContainerStyle}
```

Comment:
We don't seem to pass `hintContainerStyle` anywhere?

Reply:
Seems that get lost in some merge, I've been looking for a comment or something about why I added in the first place, but I couldn't, and can't remember I will remove it